scapy/sendrecv.py

```diff
@@ -523,8 +524,10 @@ def sendpfast(x,  # type: _PacketIterable
     else:
         argv.append("--topspeed")
 
-    if loop is not None:
-        argv.append("--loop=%i" % loop)
+    if loop:
+        argv.append("--loop=0")
+    elif count:
+        argv.append("--loop=%i" % count)
```

Comment:
Question: if one sets both `loop=1` and `count=10`, is this the expected behaviour? I **think** I'd rather have: 
```suggestion
    if count:
        argv.append("--loop=%i" % count)
    elif loop:
        argv.append("--loop=0")
```
WDYT?

Reply:
Good point, it's a better default. I might also just throw in a
```
assert not loop, "Can't use loop and count at the same time in sendpfast"
```

Reply:
Both work for me, your call!